### ISSUE TITLE
Seems to be a maximum number of different sounds able to be played

### ISSUE THREAD

**Issue by RobertFiorentino** [adamcichy/SwiftySound#37]
I'm creating a soundboard that has 112 sounds right now, however when I try to play them all one after another they stop working after the 50th sound. No matter what order I play the sounds, after 50 they stop working. Is there a max amount of sounds that can load, and if so how can I increase that limit?

**Comment by adamcichy** [comment]
Looks like a limitation of `AVFoundation`. Have you tried changing the category?

`Sound.category = .playback`

**Comment by RobertFiorentino** [comment]
No good. Many games use more than 50 sounds, would two separate Sound instances work, if I were to alter the class methods appropriately?

**Comment by adamcichy** [comment]
Try changing number of players per sound.
`Sound. playersPerSound = ...`
Default is 5.

**Comment by RobertFiorentino** [comment]
Thanks for the help! I found that raising this number actually made the problem worse. The problem is with too many players. Lowering it proportionally raised the amount of different sounds I could have. However I was able to clear the sounds that built up by setting the playersPerSound (even to the same number) because the setter clears out the sounds array. So my solution is to set playersPerSound ever so often (I just do it every time I call stopAllSounds).
